### ISSUE TITLE
Pending initiated Loop Out

### ISSUE THREAD

**Issue by mrsieb** [lightninglabs/loop#381]
```

`lit@raspberrypi:~ $ lit-loop monitor
Note: offchain cost may report as 0 after loopd restart during swap
2021-04-27T16:39:00+01:00 LOOP_OUT FAILED (FAILURE_REASON_OFFCHAIN) 0.xxx0 BTC - bc1qxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx2guj (cost: server 0, onchain 0, offchain 0)

'2021-04-28T01:17:23+01:00 LOOP_OUT INITIATED 0.xxxx5 BTC - bc1qxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx7vp7dj
2021-04-28T01:18:41+01:00 LOOP_OUT INITIATED 0.xxxx5 BTC - bc1qhxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxlex`
```

I have this stat now since 3 days nothing happens up to know. First loop out faild this is ok regular parameter mismatch.
2nd two i set a target of 400 blocks. And accidently press twice on the GUI. Target of 400 blocks ~3days, Should this not be timed out allready? Is it still ongoing ? Can i cancle it?

**Comment by carlaKC** [comment]
If your swap failed with `FAILURE_REASON_OFFCHAIN` then it means that your off-chain payment never reached the server, so your funds are still in your lnd wallet (nothing needs to be reclaimed). 

If you check `lncli listpayments --include_incomplete` is the swap payment still in an `IN_FLIGHT` state? We've recently [patched](https://github.com/lightningnetwork/lnd/pull/5214) an edge case where MPP payments can get stuck in-flight (although this is just an issue with how we store the payment on disk, all funds are retrieved). The swap is likely stuck because of that, which will be addressed in the next lnd release.

**Comment by mrsieb** [comment]
With satus IN_FLIGHT i found just 4, but a lot with satus FAILED. I just wounder if this will be removed from the Histroy in the LIT terminal. And I also dont want to bother other peers with ongoing HLTCs which not work. 

![Selection_176](https://user-images.githubusercontent.com/1194346/116808422-fa40c880-ab38-11eb-810f-fb0c8055f736.png)

**Comment by carlaKC** [comment]
> With satus IN_FLIGHT i found just 4

This output is from the loop terminal, what we need to see is the actual swap payments in `lncli listpayments` to confirm that these payments are stuck. 

> And I also dont want to bother other peers with ongoing HLTCs which not work.

If the problem is what I suspect it is, there aren't any active htlcs, it's just your lnd which hasn't properly marked the payments as failed. Nothing in flight, all funds are back in your wallet, the payment just didn't finalize its state.

**Comment by mrsieb** [comment]
Oh yes, grep the output of  `lncli listpayments --include_incomplete ` in a txt and this gets 33MB size 4 of them have status IN_FLIGHT a lot other FAILURE_REASON_NONE may i DM this to you ?

**Comment by carlaKC** [comment]
> or just the IN_FLIGHT one?

Just the `IN_FLIGHT` ones is good 👌 I'm carla on the lightning community slack if you want to drop it in dms there.

**Comment by carlaKC** [comment]
Took a look at the in-flight payments, as expected they've hit the in-flight bug in lnd. Will be resolved once lnd 0.13.0 is cut.

Thanks for the report!
